kernel/kernel-core/src/main/java/io/mosip/kernel/core/util/CryptoUtil.java

```diff
@@ -90,10 +92,35 @@ public static String encodeBase64String(byte[] data) {
 	 * @param data data to decode
 	 * @return decoded data
 	 */
+	/* This impl was a upgrade from apache coded to java 8 as apache has a single decoder  
+	 * for decoding both url safe and standard base64 encoding but java 8 has two decoders 
+	 * we are follwing this approach.
+	 */
+	@Deprecated(since = "1.1.5", forRemoval = true)
 	public static byte[] decodeBase64(String data) {
+		try {
+			return Base64.getUrlDecoder().decode(data.getBytes());
+		} catch (IllegalArgumentException exception) {
+			return Base64.getDecoder().decode(data.getBytes());
+		}
+	}
+
+	public static String base64EncodeURLSafe(byte[] data) {
+		return Base64.getUrlEncoder().encodeToString(data);
+	}
+
+	public static byte[] base64DecodeURLSafe(String data) {
 		return Base64.getUrlDecoder().decode(data.getBytes());
 	}
 
+	public static String base64Encode(byte[] data) {
+		return Base64.getEncoder().encodeToString(data);
+	}
+
+	public static byte[] base64Decode(String data) {
+		return Base64.getDecoder().decode(data.getBytes());
+	}
+

```

Comment:
Can you rename methods as below

encodeToPlainBase64
decodePlainBase64
encodeToURLSafeBase64
decodeURLSafeBase64

Reply:
done